src/http/client.rs

```diff
@@ -11,6 +11,7 @@ const MAX_ALLOWED_BYTES: usize = 20_000_000;
 pub struct Client<'app> {
     client: ActixWebClient,
     user_agent: &'static str,
+    // Why did you decide to borrow here instead of own. Was it to avoid copying?
```

Comment:
Yeah. I was working on the assumption that it was always better to borrow for better memory performance — is that the wrong approach?

Reply:
It's not "wrong", but in this particular case, I doubt the impact to memory performance is meaningful. In exchange for not having an owned struct field and avoiding a copy, you get some noise in the syntax. Not a bad trade.